src/test/java/uk/gov/hmcts/darts/task/service/impl/ManualDeletionProcessorImplTest.java

```diff
@@ -89,6 +98,10 @@ void processShouldDeleteMediaAndTranscription() {
         verify(unstructuredDeleter).delete(any(ExternalObjectDirectoryEntity.class));
         verify(logApi, times(1)).mediaDeleted(MEDIA_ID);
         verify(logApi, times(1)).transcriptionDeleted(TRANSCRIPTION_ID);
+        verify(userIdentity, times(1)).getUserAccount();
+
+        verify(mediaAction.getMedia(), times(1)).markAsDeleted(userAccount);
```

Comment:
As this test makes heavy use of mocks, it's not totally clear to me that the `deletedBy` and `deletedTimestamp` fields are being successfully updated on the associated records. 

Do you think it's worth testing this more explicitly, or is the `SoftDelete.markAsDeleted()` functionality tested elsewhere to give us confidence?

Reply:
that method auto sets the three values but agreed it maybe worth removing the mock for the SoftDelete entity to confirm exact values are set
<img width="499" alt="image" src="https://github.com/user-attachments/assets/fd937a51-e087-4a3a-9a51-112681bcdebc">

Reply:
Test updated to verify raw values instead of method call